services/algolia-sync/src/components/aws-sqs/config.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config();
+
+module.exports = {
+  receive: () => ({
+    AttributeNames: ['SentTimestamp'],
+    MaxNumberOfMessages: 1,
+    MessageAttributeNames: ['All'],
+    QueueUrl: process.env.SQS_URI,
```

Comment:
Include this in your `./env.js` file, and make it more specific please -- something like `ALGOLIA_SQS_URI` would be better.